app/src/extra/java/org/wikipedia/donate/GooglePayActivity.kt

```diff
@@ -271,8 +271,8 @@ class GooglePayActivity : BaseActivity() {
         if (requestCode == LOAD_PAYMENT_DATA_REQUEST_CODE) {
             when (resultCode) {
                 Activity.RESULT_OK -> {
-                    data?.let { intent ->
-                        PaymentData.getFromIntent(intent)?.let { paymentData ->
```

Comment:
This name of `intent` is shadowing the Activity's `intent` getter (shorthand for the function `getIntent()`), so when we get the campaign ID a few lines later, by calling `intent.getStringExtra...`, we're referencing the wrong intent!